run_gesture_tests.sh

```
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Run the magnetic gesture recognizer tests
+
+echo "================================"
+echo "Magnetic Gesture Recognizer Test"
+echo "================================"
+echo ""
+
+# Navigate to project directory
+cd /home/daniel/work/puzzgameFlutter
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)